cana/utils.py

```diff
@@ -64,6 +65,18 @@ def statenum_to_binstate(statenum, base):
 	# bstate = bin(statenum)[2:].zfill(base)
 	return bstate
 
+
+def statenum_to_output_list(statenum, base):
+	'''
```

Comment:
I guess this style is recognizable to sphinx since I copy the style from sphinx's help page. 
Yes, the basic old documentation style is not as good looking as new style. I will update them.